cdc/puller/ddl_puller.go

```diff
@@ -42,6 +49,107 @@ const (
 	DDLPullerTableName = "DDL_PULLER"
 )
 
+// DDLJobPuller is used to pull ddl job from TiKV.
+type DDLJobPuller interface {
```

Comment:
Maybe we can just implement `DDLPuller`?

Reply:
no, schemaStorage also needs `DDLJobPuller`, which is a puller. not `DDLPuller`